src/lib/db/access-store.ts

```diff
@@ -917,7 +918,9 @@ export class AccessStore implements IAccessStore {
                     SELECT r.name
                     FROM   role_user ru
                     INNER JOIN roles r on ru.role_id = r.id
-                    WHERE ru.user_id = u.id and r.type='root'
+                    WHERE ru.user_id = u.id and r.type IN (${ROOT_ROLE_TYPES.map(
+                        (type) => `'${type}'`,
+                    ).join(',')})
```

Comment:
This is our own constant, so it should be safe to inject it like this, but I'm open to suggestions.

Reply:
Seems sane to me, since we own the type